code/drasil-printers/lib/Language/Drasil/Debug/Print.hs

```diff
@@ -80,7 +80,7 @@ mkTableFromLenses pin@PI { _ckdb = db } tableLens ttle hsNEs =
     nestNum = 30
 
 openTerm :: NamedIdea a => PrintingInformation -> (String, a -> Doc)
-openTerm pinfo = ("Term", sentenceDoc (pinfo ^. ckdb) (pinfo ^. stg) MultiLine . phraseNP . view term)
+openTerm pinfo = ("Term", sentenceDoc pinfo (pinfo ^. ckdb) (pinfo ^. stg) MultiLine . phraseNP . view term)
```

Comment:
If `pinfo` is going get passed to `sentenceDoc`, there's no point in also unpacking `ckdb`, that's redundant. Same below.